### PR TITLE
Code New Branch - Working with Git Sections

### PR DESCRIPTION
**Pull request by mshajid**
Working with Git Sections

### CODE REVIEW

File: Before You Start/4. Git & GitHub Mastery.md [hunk 1]
```diff
@@ -2,47 +2,20 @@
...
-### Why 🚀 & When 📅 We Need Git In Real Life Situation? (Bob 🧔🏽 & John 👨🏽‍🦲Version)
-
-To understand Git 🤔, then we need to understand it's scenario and why we need to use them, Everyone knows that ✨ Git is a Version Control System but they don't know what are actual reasons why (including "_**when**_") we need to use Git in real life situation 🥲?
-
-Before we taking any further steps, make sure revisiting a subject or section isn't a crime. Sometimes you have to come back to the section to clarify things that won't get understandable at first glance.
-
-So we brought you up with a (🧔🏽) _**Bob &**_ (👨🏽‍🦲) _**John "GIT" Version**_.
-<br>
-<br>
+- Check here 🚀 [**Git Explained in 100 Seconds by FireShip**](https://www.youtube.com/watch?v=hwP7WQkmECE&pp=ygUQR2l0IGNyYXNoIGNvdXJzZQ==)
```
Comment (SomeOrdinaryBro) [inline Before You Start/4. Git & GitHub Mastery.md:5]: This good, but the table looked and felt like a custom thing you worked on but.. replacing with this video is also fine..

What do you think about having the table and this? Idk I just like that better

**Review by SomeOrdinaryBro — APPROVED** [review]
This is good. Let me know when you want to merge it

File: Before You Start/4. Git & GitHub Mastery.md (hunk 1 above)
Comment (mshajid) [reply Before You Start/4. Git & GitHub Mastery.md:5]: Alright let me work on that.

Comment (SomeOrdinaryBro) [reply Before You Start/4. Git & GitHub Mastery.md:5]: Cool, just a suggestion tho, add the table and below that maybe include fireship's video. like an optional thing.. 
_"if you still didnt understand what git is.. go check this video out"_

somethin like that?

File: Before You Start/4. Git & GitHub Mastery.md [hunk 2]
```diff
@@ -206,36 +179,94 @@ For this tutorial, we will install the standalone installer and explain how to i
...
+
+## How to Work With Git?
+
+Working with Git is really simple but require a deliberate practice to master Git in no time. Before hand working with Git, You've to install Git on your computer then you've to setup properly with your credentials. Check the below content for your reference to setup git. Make sure to follow in order to understand Git basics. 
+
+## Setup Git
+
+Configure your name and email: Open terminal or command prompt and set your name and email in Git. This is like telling Git who you are.
+
+`git config --global user.name "Your Name"`
```
Comment (SomeOrdinaryBro) [inline Before You Start/4. Git & GitHub Mastery.md:195]: Please Review this and un-comment it if you think this is okay to include in this tutorial. @mshajid

Comment (mshajid) [reply Before You Start/4. Git & GitHub Mastery.md:195]: Yes. we have to.

**Pull request merged**